openshift-knative-operator/pkg/common/config.go

```diff
@@ -34,3 +34,22 @@ func ConfigureIfUnset(s *base.CommonSpec, cm, key, value string) {
 	}
 	s.Config[cm][key] = value
 }
+
+// ConfigureIfUnsetAny sets a value in the config-domain if any configuration is not already set.
+// For example, the config-domain can take an arbitrary domain as a key, so it should be used here.
+func ConfigureIfUnsetAny(s *base.CommonSpec, cm, key, value string) {
```

Comment:
I had a bit a hard time to distinguish `ConfigureIfUnsetAny` from `ConfigureIfUnset` as one checks for a key and the other one checks for the whole config-map entry. What do you think about `ConfigureIfConfigmapUnset` as a name (or something like this)?